Tutorials/seL4Kernel/setting-up.md

```diff
@@ -0,0 +1,237 @@
+---
+toc: true
+layout: tutorial
+SPDX-License-Identifier: CC-BY-SA-4.0
+SPDX-FileCopyrightText: 2024 seL4 Project a Series of LF Projects, LLC.
+---
+
+# Setting up your machine
+**Overview**
+- Set up your machine - install dependencies required to run seL4``
+- Run seL4test on a simulator
+- Gain awareness of terminology used for seL4
```

Comment:
These should either be links or a paragraph. "Overview" should be a heading.

Reply:
I've removed this, since the contents list on the right provides links to each of the sections that are on the page.